www/src/components/forum/table/post-options.ts

```diff
@@ -1,32 +1,43 @@
 export const FILTER_OPTIONS = {
   label: "Filter by category",
   options: [
-    { label: "All", value: "all" },
-    { label: "Delegates", value: 41 },
-    { label: "General Discussions", value: 1 },
-    { label: "Mission Grants", value: 69 },
-    { label: "Updates and Announcements", value: 48 },
-    { label: "Retro Funding", value: 46 },
-    { label: "Others", value: "others" },
+    { label: "All", value: "all", slug: "all" },
+    { label: "Delegates", value: "41", slug: "delegates" },
+    { label: "General Discussions", value: "1", slug: "general" },
+    { label: "Mission Grants", value: "69", slug: "grants" },
+    { label: "Updates and Announcements", value: "48", slug: "updates" },
+    { label: "Retro Funding", value: "46", slug: "retro-funding" },
+    { label: "Others", value: "others", slug: "others" },
   ],
 };
 
+export const categoryBySlug = (slug: string) => {
+  return FILTER_OPTIONS.options.find((option) => option.slug === slug)?.value;
+};
```

Comment:
to avoid calling .find every time
```suggestion
export const CATEGORY_BY_SLUG: { [key: string]: string } = FILTER_OPTIONS.options.reduce(
  (acc, option) => ({
    ...acc,
    [option.slug]: option.value,
  }),
  {}
);

export const categoryBySlug = (slug: string): string => CATEGORY_BY_SLUG[slug] || "";
```